addons/website_event_track/views/event_track_templates.xml

```diff
@@ -376,6 +376,12 @@
                                     <textarea  class="form-control" name="biography" rows="5"/>
                                 </div>
                             </div>
+                            <div class="form-group row">
```

Comment:
Side note: The name of the tab in my browser is "event_track_proposal".
Could be nice if we can do something about it.

![Selection_357](https://user-images.githubusercontent.com/41282536/72607977-f7a96800-3921-11ea-951c-18cee8b16819.png)